src/components/Atoms/Icon/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SvgIcon } from '@mui/material';
+import { ICONS } from '../../../utils/Constants';
+
+export interface IconPropsType {
+    name:
+      | 'home'
+      | 'clientManagement'
+      | 'configurations'
+      | 'accessManagement'
+      | 'orchesstration'
+      | 'riskCaseManagement'
+      | 'reports';
+    [key: string]: unknown;
+  }
```

Comment:
No need of mentioning it like this right ? It can be any icon, just send whatever icon you want as prop, instead of sending name